### PR TITLE
Clarify errors in ProjectedVolume validation

### PR DESCRIPTION
**Pull request by thockin**
Seen while reviewing other things.  Just some string cleanups.

/kind cleanup

```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/api/core/v1/types.go [hunk 1]
```diff
@@ -1911,7 +1911,8 @@ type ClusterTrustBundleProjection struct {
 
 // Represents a projected volume source
 type ProjectedVolumeSource struct {
-	// sources is the list of volume projections
+	// sources is the list of volume projections.  Each entry in this list
```
Comment (aojea) [inline staging/src/k8s.io/api/core/v1/types.go:1914]:
```suggestion
	// sources is the list of volume projections. Each entry in this list
```

**Comment by thockin** [comment]
all green now

**Comment by thockin** [comment]
ping

**Comment by thockin** [comment]
one more ping - this is a simple PR I'd like to take off my board :)

**Pull request merged**